### ISSUE TITLE
Chat on google hangout ?

### ISSUE THREAD

**Issue by Carreau** [fabianp/nbgallery#1]
Please, please please, 

Come chat on google hangout with the the rest of IPython/Jupyter team, 
we are looking to do something [like what you did](http://nb.bianp.net/sort/views/)  on Nbviewer
for a loooooong time, and did not found time or people to do it !

You could even hooked directly in nbviewer statistics if needed.

@rgbkrk and @bollwyvl you got the key to nbviewer,  what do you think ?

**Comment by rgbkrk** [comment]
THATS AWESOME! Excellent work.

**Comment by bollwyvl** [comment]
hooray, great concept! :ice_cream:

Let's indeed talk some more!

**Comment by Carreau** [comment]
Actually can we ping a full team like @ipython / @jupyter ?

**Comment by fabianp** [comment]
Hi and thanks for your kind words!

Of course I'd be glad to have a chat and talk about how we can collaborate better. Any time would be good for me.

**Comment by Carreau** [comment]
> Hi and thanks for your kind words!
> 
> Of course I'd be glad to have a chat and talk about how we can collaborate better. Any time would be good for me.

There are meeting on tuesday 9:30 PST (that 18h30 in Paris) jump on gitter.im/ipython/ipython and you Should get a link to join. 

I won't join today as am flying tomorrow morning from Paris to Berkeley, but will catch up when the meeting is on youtube later.

**Comment by fabianp** [comment]
I'll be glad to join today. See (some of you) later then

**Comment by rgbkrk** [comment]
Yeah... I won't be there today or Thursday for family stuff.

**Comment by fabianp** [comment]
Is it better if we chat next week ? For me it's the same

**Comment by Carreau** [comment]
It's same time every week, so it's whenever you like. 
The agenda is here : https://ipython.hackpad.com/
So feel free to move things around, look ad what the discussion subject will be...

I think Fernando and Brian are still locked up in the basement to write their grant,
so next week might be better also, but I'm not sure.

**Comment by rgbkrk** [comment]
Yeah, next week is better for me though the next few weeks will be 10 shades of crazy.

**Comment by fabianp** [comment]
I was today in the Hangouts. It was a good thing to speak directly with some of the IPython devs. What I take from the meeting is that:
- For now it's not a bad idea to have nbviewer and a gallery as separate apps.
- We will see in a couple of months where both projects are: IPython devs want to focus with ipython 3.0 and for me I still need time to implement basic features for the gallery.

**Comment by rgbkrk** [comment]
Sounding pretty good. We were intending to actually index content as it came in or changed as well as tally views. This got put on hold for a bit, but I'm definitely open to talking about where to take this. There are some blockers in our view about making sure to acknowledge robots.txt (since we're effectively an aggregator not disallowing anything). Really excited to hear about all this and enable you as best we can!

**Comment by zavg** [comment]
Hi guys! Please feel free to check out new service to hold regular online meetups using Google Hangouts on Air: [webmeetups.net](webmeetups.net).

The service automatically reminds about meetup start in a day and in an hour, deals with time zones, allows you to select active participants and watchers if there are more then 10 attending people. You get rid of Hangouts url and Youtube url copy-pasting, it automatically handles all these things. Also it nicely embeds all YouTube videos in your meetup history.

I am service founder and we just came to production, so I warmly invite you to check out our service. Do not hesitate to ask any questions!
